project/manager.go

```diff
@@ -71,6 +75,39 @@ func (m *Manager) load(projectID uint64) (*Project, error) {
 	return p, nil
 }
 
+func (m *Manager) loadFromLocal(projectFileDir string) error {
+	files, err := os.ReadDir(projectFileDir)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			return errors.Wrapf(err, "failed to read project directory %s", projectFileDir)
+		}
+	}
+	for _, f := range files {
+		if f.IsDir() {
+			continue
+		}
+		data, err := os.ReadFile(path.Join(projectFileDir, f.Name()))
+		if err != nil {
+			slog.Error("failed to read project config", "filename", f.Name(), "error", err)
```

Comment:
"failed to read project config" -> "failed to read project file"

Reply:
fix in another pr